src/styles/main.scss

```diff
@@ -2,9 +2,9 @@
 @import "dropdown";
 @import "searchbox";
 
-$searchbox: true!default;
+$searchbox: true !default;
```

Comment:
I'm a bit rusty on scss syntax, but would those spaces change the meaning or is it ok?

Reply:
It is ok

Reply:
:+1: